virtualizarr/tests/test_readers/test_netcdf3.py

```diff
@@ -1,15 +1,15 @@
-import numpy as np
+import pytest
 import xarray as xr
 import xarray.testing as xrt
 
 from virtualizarr import open_virtual_dataset
 from virtualizarr.manifests import ChunkManifest, ManifestArray
 from virtualizarr.tests import requires_scipy
-from virtualizarr.zarr import ZArray
 
 
 @requires_scipy
-def test_read_netcdf3(netcdf3_file):
+@pytest.mark.xfail(reason="zarr-python 3.0 does not support big endian")
```

Comment:
I don't think the netcdf3 reader will work with ArrayV3Metadata because the data type is big endian, which is not yet supported by zarr-python 3. See [v3 migration guide](https://zarr.readthedocs.io/en/stable/user-guide/v3_migration.html):

>The following features that were supported by Zarr-Python 2 have not been ported to Zarr-Python 3 yet:
>- Big Endian dtypes ([#2324](https://github.com/zarr-developers/zarr-python/issues/2324))